server/test/api/functions.js

```diff
@@ -0,0 +1,94 @@
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+var uuid    = require('node-uuid');
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+/**
+* The /function  API endpoint
+*
+* This test suite implements full CRUD on the /function  HTTP API endpoint.
+*/
+describe('The /function  API endpoint', function () {
```

Comment:
@lomamech , it is critical that you update this string.  Please update both the string and the comments.  We rely on them to tell us what is happening in out debugging procedures.